server/game/cards/03-WC/Malison.js

```diff
@@ -0,0 +1,17 @@
+const Card = require('../../Card.js');
+
+class Malison extends Card {
+    setupCardAbilities(ability) {
+        this.fight({
+            target: context => ({
```

Comment:
You cannot pass a function to `target` (and I'm not sure why you are trying to here....)

Reply:
Oops, this wasn't meant to get checked in!